### PR TITLE
Skip CT for master

### PR DESCRIPTION
**Pull request by vkatsuba**
Propose skip CT for master for save time for build images

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -75,13 +79,6 @@ jobs:
         - docker push $BUILD_IMAGE:$TAG
         - echo "docker push done"
 
-addons:
-  artifacts:
```
Comment (vkatsuba) [inline .travis.yml:79]: If CT skip for master, no make sense upload CT logs cuz artifacts disabled for PR's

**Comment by RoadRunnr** [comment]
I'm not convince that this is a good idea.

Also, WHY? It's not like anyone is impatiently waiting for the CI to pass.

**Comment by vkatsuba** [comment]
> Also, WHY? It's not like anyone is impatiently waiting for the CI to pass.

The main reason that the image can build only after CT pass and this mean if we need to start testing we need to await +-  1 hour, eg if we have 2 fixes we need to await 2 hours for build image from master instead of +- 15 min, second point is, basically we run test twice instead of only one only  for PR.

**Comment by RoadRunnr** [comment]
> > Also, WHY? It's not like anyone is impatiently waiting for the CI to pass.
> 
> The main reason that the image can build only after CT pass and this mean if we need to start testing we need to await +- 1 hour, eg if we have 2 fixes we need to await 2 hours for build image from master instead of +- 15 min, second point is, basically we run test twice instead of only one only for PR.

But travis pushes to docker hub and we should be using quay.io for testing anyhow. I wouldn't mind abandoning docker hub altogether.

Building images for non-tag, master builds with expiry label set could be done before testing is complete. There is no real reason to wait for the tests to finish.
Images with expire label should probably still wait for the test to finish to avoid polluting the registry with broken images.

**Comment by vkatsuba** [comment]
Closed cuz current implementation is not suit.

**Pull request closed without merging**